sentinelsat/sentinel.py

```diff
@@ -1438,3 +1438,40 @@ def _parse_odata_response(product):
                 pass
         output[attr["Name"]] = value
     return output
+
+
+def placename_to_wkt(placename):
+    """Geocodes the placename to rectangular bounding extents using Nominatim API and
+       returns the corresponding 'ENVELOPE' form Well-Known-Text.
+
+    Parameters
+    ----------
+    placename : str
+        the query to geocode
+
+    Returns
+    -------
+    full name and coordinates of the queried placename
+        list in the form [string placeinfo, wkt string in form 'ENVELOPE(minX, maxX, maxY, minY)']
+    """
+
+    rqst = requests.post(
+        "https://nominatim.openstreetmap.org/search", params={"q": placename, "format": "geojson"}
+    )
+    # Check that the response from Openstreetmapserver has status code 2xx
+    # and that the response is valid JSON.
+    _check_scihub_response(rqst)
```

Comment:
Should simply use `rqst.raise_for_status()` here since the extra error handling included in `_check_scihub_response()` is not relevant for Nominatim.